src/ProgressOnderwijsUtils/Data/DbLoadingHelperImpl.cs

```diff
@@ -8,7 +8,13 @@ public static class DbLoadingHelperImpl
         //called via reflection from DataReaderSpecialization
         public static byte[] GetBytes(this IDataRecord row, int colIndex)
         {
-            var byteCount = row.GetBytes(colIndex, 0L, null, 0, 0);
+            var byteCount = row.GetBytes(
+                colIndex,
+                0L,
+                null!, // Incorrect annotation? https://github.com/dotnet/runtime/blob/35e535e6fa50b8284a44428d6b03b0180bc9b499/src/libraries/System.Data.Common/src/System/Data/Common/DataRecordInternal.cs#L123

```

Comment:
schiet jij een issue in? Of ben je wel uit ge-issued?

Reply:
https://github.com/dotnet/runtime/issues/44886

Reply:
Wat ook opmerkelijk is aan dit hele verhaal: ze lijken geen runtime analysis te hebben gedaan van nullability.  Je zou toch denken dat als je echte code draait, je allemaal van dit soort gevallen triviaal had kunnen vangen. Zelfs de compiler had mogelijk deze issue kunnen vangen als ze dit breed hadden ge-dog-food, want lijkt me vreemd dat ze dit zelf niet ergens gebruiken. Ze hebben zelf dus ook geen betere manier om nullability te annoteren dan wat wij doen: prutsen en kijken wat gebeurt.